osf/utils/migrations.py

```diff
@@ -407,7 +407,7 @@ def create_schema_blocks_for_atomic_schema(schema):
         block['schema_block_group_key'] = ''
         block_type = block['block_type']
 
-        if block_type == 'question-label':
+        if block_type in ('question-label', 'paragraph'):
```

Comment:
paragraph doesn't need to generate its own group key -- that would create a split between the question's group key and the input blocks' group key if a paragraph follows the label for some reason.